### PR TITLE
docs: Clarify "dummy data"

### LINKED ISSUE #1570: Clarify *dummy data*
Use _dummy tables_ in place of _dummy (input) data_ and _dummy dataset_ in place of _dummy (output) data_. Discussed and agreed during today's data team meeting.

Follows @inglesp's comment on #1551:

> The only other bit of feedback I have is about dummy input data and dummy output data. This is a distinction that makes most sense when explaining the differences in dummy data between cohort-extractor and ehrQL. Could we get away with just talking about "dummy data" in the ehrQL docs?

### PR DESCRIPTION
**Pull request by iaindillingham**
Closes #1570

### CODE REVIEW

File: docs/tutorial/working-with-frames-and-series/index.md [hunk 1]
```diff
@@ -113,13 +113,17 @@ The `medications.date` column is an event series; it has many rows per patient.
 9 | 9 | 2015-03-14
 ```
 
-## Transform input data into output data
+## Transform tables into a dataset
 
-ehrQL tables, such as `patients` and `medications`, contain input data.
+Tables, such as `patients` and `medications`, contain either real or dummy level two data.
```
Comment (inglesp) [inline docs/tutorial/working-with-frames-and-series/index.md:118]: Is it a distraction to introduce the security levels here?

Comment (inglesp) [reply docs/tutorial/working-with-frames-and-series/index.md:118]: If we do talk about them, we should call them eg "Level 2" for consistency elsewhere.

File: ehrql/__main__.py [hunk 2]
```diff
@@ -208,18 +208,18 @@ def add_create_dummy_tables(subparsers, environ, user_args):
         "create-dummy-tables",
         help=strip_indent(
             """
-            Generate dummy data for a dataset and write out tables as CSV.
+            Generate dummy tables and write them out as CSV files (one per table).
```
Comment (inglesp) [inline ehrql/__main__.py:211]: nit: do you write _to_ CSV files or _as_ CSV files?

**Review by inglesp — APPROVED** [review]
[Real tables for my dummy friends!](https://xkcd.com/1645/)

File: ehrql/__main__.py (hunk 2 above)
Comment (iaindillingham) [reply ehrql/__main__.py:211]: I don't know, but previously we used "as". Do you have a preference?

File: docs/tutorial/working-with-frames-and-series/index.md (hunk 1 above)
Comment (iaindillingham) [reply docs/tutorial/working-with-frames-and-series/index.md]: Changed.

File: ehrql/__main__.py (hunk 2 above)
Comment (iaindillingham) [reply ehrql/__main__.py:211]: Not changed.

File: docs/tutorial/working-with-frames-and-series/index.md (hunk 1 above)
Comment (inglesp) [reply docs/tutorial/working-with-frames-and-series/index.md]: This is clearer, thanks.

File: ehrql/__main__.py (hunk 2 above)
Comment (inglesp) [reply ehrql/__main__.py:211]: Fine :)

**Pull request merged**